contracts/TablelandTables.sol

```diff
@@ -51,28 +51,97 @@ contract TablelandTables is
     }
 
     /**
-     * @dev See {ITablelandTables-createTable}.
+     * @custom:deprecated See {ITablelandTables-createTable}.
+     * This function is deprecated, please use `create`.
      */
     function createTable(
         address owner,
-        string memory statement
-    ) external payable override whenNotPaused returns (uint256 tableId) {
-        tableId = _nextTokenId();
-        _safeMint(owner, 1);
+        string calldata statement
+    ) external payable override whenNotPaused returns (uint256) {
```

Comment:
Honestly, I don't know the best path to removal of a method.  I'll give that a try.

Reply:
@sanderpick I'm trying to work through this and I the stumbling block I'm hitting is that our tests for reentrancy in the deprecated methods don't work anymore.  This is because the test contracts create an `ITablelandTables` instance based on an existing contract, and since the interface doesn't match the contract this is no longer possible.

refs:
https://github.com/tablelandnetwork/evm-tableland/pull/257/files#r1152506868 
https://github.com/tablelandnetwork/evm-tableland/pull/257/files#r1152507967

Reply:
cool, i don't have a strong opinion on dropping the methods from the interface. I think it would be cleaner, but if it's causing a headache, we can leave it.

Reply:
I'll spend a little more time trying to figure this out. I'm not sure how to interact with the registry contract from another contract if the registry contract interface is different from the registry contract, do you know how that works in Solidity?

Reply:
Can you just do ` _tableland = TablelandTables(registry);`, ie, use the contract and not the interface?

Reply:
Duh! yep that's perfect